pkg/kernel/networkservice/ethernetcontext/vf_client.go

```diff
@@ -1,8 +1,8 @@
-// Copyright (c) 2022 Cisco and/or its affiliates.
+// Copyright (c) 2022-2023 Cisco and/or its affiliates.
 //
-// Copyright (c) 2021-2022 Nordix Foundation.
+// Copyright (c) 2021-2023 Nordix Foundation.
 //
-// Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
+// Copyright (c) 2021-2023 Doc.ai and/or its affiliates.
```

Comment:
Note: you could move Cisco to the bottom and change only it

Reply:
Thanks, fixed